puppet/modules/quickstack/manifests/neutron/all.pp

```diff
@@ -208,14 +237,21 @@
       tenant_network_type          => $tenant_network_type,
     }
   } else {
-    $local_ip = find_ip("$ovs_tunnel_network",
+    if ('cisco_n1kv' in $ml2_mechanism_drivers) {
+      $_enable_tunneling  = str2bool_i("false")
+      $local_ip = "false"
```

Comment:
Minor note, I would suggest you make these:
```
$_enable_tunneling = false
$local_ip = false
```

'false' as a string evaluates to true in puppet, so local_i may not resolve as you expect when using the n1kv driver, the _enable_tunneling change is just a simplification.  We only call str2bool_i to guarantee it is a boolean that was passed in by the user, but you control that type here, so it is unnecessary to call the function here.